publisher-sdk/src/test/java/com/criteo/publisher/util/jsonadapter/BooleanJsonAdapterTest.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher.util.jsonadapter
+
+import com.squareup.moshi.JsonDataException
+import org.assertj.core.api.Assertions
+import org.junit.Before
+import org.junit.Test
+
+class BooleanJsonAdapterTest {
+
+  private lateinit var adapter: BooleanJsonAdapter
```

Comment:
I would rather test from the provided mochi object rather than on the adapter itself. For sure testing the adapter directly is more unitary, but from SDK point of view, mochi (with the adapter) is used, not the adapter itself